src/assets/i18n/en.json5

```diff
@@ -5639,4 +5639,332 @@
   "admin.notifications.publicationclaim.breadcrumbs": "Publication Claim",
 
   "admin.notifications.publicationclaim.page.title": "Publication Claim",
+
+  "coar-notify-support.title": "COAR Notify Protocol",
+
+  "coar-notify-support-title.content": "Here, we fully support the COAR Notify protocol, which is designed to enhance the communication between repositories. To learn more about the COAR Notify protocol, you can visit their official website <a href=\"https://notify.coar-repositories.org/\">here</a>.",
```

Comment:
We should never add a link that says "here" as that's bad for accessibility.  This last sentence should be changed to:
```
To learn more about the COAR Notify protocol, visit the <a href=\"https://notify.coar-repositories.org/\">COAR Notify website</a>.
```